conda/recipes/dask-cuda/meta.yaml

```diff
@@ -24,11 +24,12 @@ requirements:
     - setuptools
   run:
     - python x.x
-    - dask-core >=2.4.0
+    - dask >=2.4.0
```

Comment:
It appears we are using `dask` here. Am I missing something else?

Reply:
We are now, after Mads' changes in https://github.com/rapidsai/dask-cuda/pull/353/commits/3756b0a61a23dbce95288664e4e11e2e5e79661b, which wasn't the case when we were having CI issues.